packages/webpack-cli/lib/utils/index.js

```diff
@@ -1,6 +1,22 @@
+let colorette = null;
+
 module.exports = {
+
   get colors() {
-    return require("colorette");
+    if (!colorette || colorette.options.changed) {
+      const col = require('colorette');
+      const options = {
+        changed: false,
+        enabled: colorette? colorette.options.enabled : true,
```

Comment:
It seems some weird, colorette already has logic for `enabled`, why we need check `colorette`?

Reply:
See existing code at https://github.com/webpack/webpack-cli/blob/3813e7ae22b22be63c84d530072073254bc2104a/packages/webpack-cli/lib/webpack-cli.js#L995
From my understanding this is an existing solution to trigger changes in the behaviour of instance, returned by getter. Alternatively, we can expose a method that would instantly recreate instance, stored here, to avoid this.

Reply:
We should rewrite logic, because we already cache module, we can keep `changed` on object level

Reply:
caching module is not enough, we need to store initialized coloretto instance as well